concordium-node/macos_log_wrapper.c

```diff
@@ -0,0 +1,5 @@
+#include <os/log.h>
+
+void wrapped_os_log_with_type(os_log_t log, os_log_type_t type, const char* message) {
+    os_log_with_type(log, type, "%{public}s", message);
+}
```

Comment:
Please document the reason for this (the fact that os_log_with_type is a macro, so cannot be used in FFI if I recall correctly).